microsoftgraph/docs-conceptual/how-to-manage-pim-policies.md

```diff
@@ -0,0 +1,182 @@
+$policyId = "Directory_06f6521d-c18c-460a-8656-fa82e81aa94b_81328d30-ae8f-48cc-b242-48f50dd77d03_9b895d92-2cd3-44c7-9d02-a6ac2d5ea5c3"
+
+Get-MgPolicyRoleManagementPolicyAssignment -UnifiedRoleManagementPolicyAssignmentId $policyId
+```
+
+```Output
+Id                                                                                                                       PolicyId                                                                            RoleDefinitionId                     ScopeId ScopeType
+--                                                                                                                       --------                                                                            ----------------                     ------- ---------
+Directory_06f6521d-c18c-460a-8656-fa82e81aa94b_81328d30-ae8f-48cc-b242-48f50dd77d03_9b895d92-2cd3-44c7-9d02-a6ac2d5ea5c3 Directory_06f6521d-c18c-460a-8656-fa82e81aa94b_81328d30-ae8f-48cc-b242-48f50dd77d03 9b895d92-2cd3-44c7-9d02-a6ac2d5ea5c3 /       Directory
+```
+
+## Step 3: Get policy rules
+
+### Get all rules for a policy
+
+To get all the rules for a policy, run:
+
+```powershell
+$policyId = "Directory_06f6521d-c18c-460a-8656-fa82e81aa94b_81328d30-ae8f-48cc-b242-48f50dd77d03"
+Get-MgPolicyRoleManagementPolicyRule -UnifiedRoleManagementPolicyId $policyId | Format-List
+```
+
+```Output
+
+Id                   : Expiration_Admin_Eligibility
+Target               : Microsoft.Graph.PowerShell.Models.MicrosoftGraphUnifiedRoleManagementPolicyRuleTarget
+AdditionalProperties : {[@odata.type, #microsoft.graph.unifiedRoleManagementPolicyExpirationRule], [isExpirationRequired, False], [maximumDuration, P365D]}
+
+Id                   : Enablement_Admin_Eligibility
+Target               : Microsoft.Graph.PowerShell.Models.MicrosoftGraphUnifiedRoleManagementPolicyRuleTarget
+AdditionalProperties : {[@odata.type, #microsoft.graph.unifiedRoleManagementPolicyEnablementRule], [enabledRules, System.Object[]]}
+```
+
+### Get  a specific rule for a policy
+
+To get the details of a specific rule for a policy, run:
+
+```powershell
+$policyId = "Directory_06f6521d-c18c-460a-8656-fa82e81aa94b_81328d30-ae8f-48cc-b242-48f50dd77d03"
+
+$ruleId = "Expiration_EndUser_Assignment"
+
+Get-MgPolicyRoleManagementPolicyRule -UnifiedRoleManagementPolicyId $policyId -UnifiedRoleManagementPolicyRuleId $ruleId | Format-List 
+```
+
+```Output
+
+Id                   : Expiration_EndUser_Assignment
+Target               : Microsoft.Graph.PowerShell.Models.MicrosoftGraphUnifiedRoleManagementPolicyRuleTarget
+AdditionalProperties : {[@odata.context, https://graph.microsoft.com/v1.0/$metadata#policies/roleManagementPolicies('Directory_06f6521d-c18c-460a-8656-fa82e81aa94b_81328d30-ae8f-48cc-b242-48f50dd77d03')/rules/$entity], [@odata.type, #microsoft.graph.unifiedRoleManagementPolicyExpirationRule],
+                       [isExpirationRequired, True], [maximumDuration, PT8H]}
+```
+
+## Step 4: Update a policy rule 
+
```

Comment:
Is this just an example, and the same steps should be followed for other rules they need to update? We could perhaps express that more clearly if that is the case